publish-service/src/main/java/com/ericsson/eiffel/remrem/publish/config/SecurityConfig.java

```diff
@@ -64,6 +95,9 @@ public class SecurityConfig extends WebSecurityConfigurerAdapter {
     @Value("${activedirectory.connectionTimeOut:#{127000}}")
     private Integer ldapTimeOut = DEFAULT_LDAP_CONNECTION_TIMEOUT;
 
+    @Value("${LdapCacheTTL}")
```

Comment:
The property should be part of configuration file (including appropriate description).